proto/storage/node.proto

```diff
@@ -84,16 +84,17 @@ message NodeScan {
 }
 
 message NodeInventory {
-    string                         node_name = 1;
-    google.protobuf.Timestamp      scan_time = 2;
+    string                           node_id = 1;
+    string                         node_name = 2;
+    google.protobuf.Timestamp      scan_time = 3;
```

Comment:
No need for backwards compatibility here as this object is fully protected by the feature flag and has not bee used by our customers before